docs/quickstart_guide/index.rst

```diff
@@ -0,0 +1,245 @@
+
+You can see which devices are available by just executing Lewis without parameters:
+
+::
+
+    $ lewis
+    Please specify a device to simulate. The following devices are available:
+        julabo
+        chopper
+        linkam_t95
+
+Some additional, simpler examples are located in the ``lewis.examples`` module. You can tell Lewis which module to scan for devices using the ``-k`` parameter:
+
+::
+
+    $ lewis -k lewis.examples
+    Please specify a device to simulate. The following devices are available:
+        dual_device
+        simple_device
+        timeout_device
+        modbus_device
+        example_motor
+
+For this guide, we will launch the example_motor:
+
+::
+
+    $ lewis -r :10000 -k lewis.examples example_motor
+    INFO lewis.DeviceBase: Creating device, setting up state machine
+    INFO lewis.Simulation: Changed cycle delay to 0.1
+    INFO lewis.Simulation: Changed speed to 1.0
+    INFO lewis.Simulation: Starting simulation
+    INFO lewis.ControlServer: Listening on 0.0.0.0:10000
+    INFO lewis.AdapterCollection: Connecting device interface for protocol 'stream'
+    INFO lewis.ExampleMotorStreamInterface.StreamServer: Listening on 0.0.0.0:9999
+
+The example motor is a TCP Stream device and listens on port 9999 on all adapters by default.
+
+The ``-r`` parameter is optional and specifies an ``adapter:port`` pair for the "Control API" to listen on. Here it listens on port 10000 on all adapters. We will see more on that in following sections.
+
+
+Connect to Motor via Telnet
+===========================
+
+With the last command from the previous section still running, open another terminal window.
+
+Since the example motor conveniently uses CRLF line terminators, we can use telnet to talk to it:
+
+::
+
+    $ telnet localhost 9999
+    Trying 127.0.0.1...
+    Connected to localhost.
+    Escape character is '^]'.
+
+You're now connected to the TCP Stream interface of the example motor device. It understands the following commands:
+
+=======  =============
+Command  Meaning
+=======  =============
+S?       get status
+P?       get position
+T?       get target
+T=%f     set target
+H        stop movement
+=======  =============
+
+Note that they are case sensitive. Try entering a few commands in the Telnet session:
+
+::
+
+    S?
+    idle
+    P?
+    0.0
+    T=20.0
+    T=20.0
+    S?
+    moving
+    P?
+    9.106584
+
+See `the source code <https://github.com/DMSC-Instrument-Data/lewis/blob/master/src/lewis/examples/example_motor/__init__.py>`_ of the example motor if you want to see what makes it tick.
+
+
+Connect to Motor via Control Client
+===================================
+
+In addition to the simulated TCP Stream interface, Lewis provides a so-called Control Server interface, which allows you to bypass the normal device protocol and access both device and simulation parameters directly while the simulation is running. This can be very useful to debugging and diagnostics.
+
```

Comment:
"Think of this as a back-door for altering your simulation at runtime without making permanent changes to the usual front door interface"  (Or something similar)?

Reply:
I never really cared for the "back-door" terminology; it feels a bit too colloquial... so I'm avoiding it on purpose here.

Perhaps I can add something along the lines of "... to keep the main device interface clean" to the last sentence instead?